ui/src/reducers/draftItem.js

```diff
@@ -166,11 +166,14 @@ export default function draftsReducer(state = initialState, action) {
 
     case draftItemActions.GENERAL_TITLE_REQUEST:
       return state.set("generalTitleLoading", true);
-    case draftItemActions.GENERAL_TITLE_SUCCESS:
+    case draftItemActions.GENERAL_TITLE_SUCCESS: {
+      let current = state.get("metadata");
+      current.general_title = action.draft.metadata.general_title;
       return state
-        .set("formData", action.draft.metadata)
+        .set("metadata", current)
```

Comment:
can be removed since it is done with the `merge` under. It is setting the `formData` that was creating the issue (here `.set("formData", action.draft.metadata)`)

Reply:
you are right. Deleted the `set` and left only the `merge`